File: DAL/DbEntities/Room.cs
﻿namespace DAL.DbEntities
{
    public class Room
    {
        public int Id { get; set; }
        // TODO : add location
        public int Capacity { get; set; }
        public bool HasSpeaker { get; set; }
        public bool HasComputer { get; set; }

        public override string ToString()
        {
            // TODO : Add location
            return $"{nameof(Id)}: {Id}, {nameof(Capacity)}: {Capacity}";
        }
    }
}

File: NgClient/src/app/model/room.ts
import { RoomLocation } from './room-location';
import { RoomDto } from '../common/dto/room.dto';

export class Room {
	public constructor(roomDto: RoomDto) {
		this.description = roomDto.description;
		this.capacity = roomDto.capacity;
		this.hasSpeaker = roomDto.hasSpeaker;
		this.hasComputer = roomDto.hasComputer;
		this.location = roomDto.location;
	}

	description: string;
	capacity: number;
	hasSpeaker: boolean;
	hasComputer: boolean;
	location: RoomLocation;
}